Branch-SDK/Branch.h

```diff
@@ -683,6 +683,17 @@ typedef NS_ENUM(NSUInteger, BranchCreditHistoryOrder) {
  */
 - (void)setAppClipAppGroup:(NSString *)appGroup;
 
+/**
+ Pass the AppTrackingTransparency authorization status to Branch to measure ATT prompt performance.
+ This method should be called from the callback of ATTrackingManager.requestTrackingAuthorization.
+ 
+ Note:
+ Before prompting the user, check that ATTrackingManager.trackingAuthorizationStatus is notDetermined.
+ Otherwise the prompt will not display and the completion will be called with current status.
+ This will inflate the number of OPT_IN and OPT_OUT events tracked by Branch.
+ */
+- (void)handleOptInStatus:(unsigned long)status;
```

Comment:
Did you think of making this an `NSUInteger`? I think that would make it more transparent to Swift, as a `UInt` without casting.

Reply:
Good point, I'll take a look at that.